src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -0,0 +1,105 @@
+<Meta title="SDK Providers/VoiceFocusProvider" />
+
+# VoiceFocusProvider
+
+The `VoiceFocusProvider` provides a function transforming a normal device to an Amazon voice focus device, and the support state of voice focus status.
```

Comment:
Should we change "the support state of voice focus status" to "whether the Amazon voice focus is supported"?

Reply:
updated